ui/src/main/java/com/doordeck/sdk/common/manager/Doordeck.kt

```diff
@@ -316,8 +316,8 @@ object Doordeck {
     /**
      * Shows the verification screen
      */
-    fun showVerificationScreen(context: Context) {
-        VerifyDeviceActivity.start(context)
+    fun showVerificationScreen(context: Context, shouldUnlockAfterSuccessVerifying: Boolean) {
```

Comment:
Do we need to add a default value to `shouldUnlockAfterSuccessVerifying` as well? I don't know if we're expecting developers to call this directly or not, hence how much it affects upgrading.

Reply:
It has it indeed, but at `VerifyDeviceActivity.start`

This method is also public so I'm pushing it now